spec/03-plugins/11-basic-auth/03-access_spec.lua

```diff
@@ -4,7 +4,9 @@ local meta = require "kong.meta"
 local utils = require "kong.tools.utils"
 
 describe("Plugin: basic-auth (access)", function()
+
```

Comment:
are these newlines necessary? we could fix style here in a separate commit. these changes are outside the scope of this PR, dirtying the commit history.